src/st/wrp_psp/stage_name_new.c

```diff
@@ -1,83 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0-or-later
-#include "../wrp/wrp.h"
-
-void func_8928694(s32, void*);
-void func_89286CC(s8[], void*);
-void func_8928624(s32, void*);
-void func_892865C(s32, void*);
-void func_892752C(void*, s32[]);
-
-void func_psp_09244020(Primitive* arg0) {
```

Comment:
This was just `UnkPrimHelper` so ended up being able to remove this and fully code share with PSX WRP